packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -1637,11 +1637,10 @@ export const ExplorerProvider: FC<
     useEffect(() => {
         if (!state.shouldFetchResults) return;
 
-        mutateAsync().then(() => {
-            dispatch({
-                type: ActionType.SET_FETCH_RESULTS_FALSE,
-            });
-        });
+        (async () => {
```

Comment:
Ditto on using a named function or closure and avoiding the short form here, for readability.